src/Microsoft.AspNetCore.SignalR.Core/SerializedHubMessage.cs

```diff
@@ -18,8 +18,15 @@ public class SerializedHubMessage
         private IList<SerializedMessage> _cachedItems;
```

Comment:
Up on the class-level comment. Now that this isn't in `.Internal`, it's not subject to breaking changes, so we should remove that.

Reply:
Let's just remove the first line blurb about this API being internal. It's not internal any more, so I guess we'd better just suck it up and support it :)